frontend/src/components/App/AppSidebar/RerollButton.jsx

```diff
@@ -33,27 +33,27 @@ class RerollButton extends PureComponent {
 
     render() {
         let users,
-            skips;
+            rerolls;
 
-        if (this.props.connectedUser !== null && this.props.connectedUser !== undefined) {
+        if (this.props.connectedUser) {
             users = this.props.connectedUser.length + 1;
-            skips = 0;
+            rerolls = 0;
             for (let j = 0; j < this.props.connectedUser.length; j++) {
```

Comment:
I prefer using array methods over for loops, can we update them here as well maybe?

Reply:
```
this.props.connectedUser.forEach((value, index, array) => {
                if (value.voted === 'reroll') {
                    rerolls++;
                }
            });
```
Is this what you mean